src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -90,23 +97,32 @@ public boolean equals(Object other) {
 
         return guiSettings.equals(o.guiSettings)
                 && addressBookFilePath.equals(o.addressBookFilePath)
+
                 && salesBookFilePath.equals(o.salesBookFilePath)
+
                 && ingredientBookFilePath.equals(o.ingredientBookFilePath);
     }
 
     @Override
     public int hashCode() {
+
         return Objects.hash(guiSettings, addressBookFilePath, salesBookFilePath, ingredientBookFilePath);
+
     }
 
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder();
         sb.append("Gui Settings : " + guiSettings);
         sb.append("\nLocal data file location : " + addressBookFilePath);
+
+        sb.append("\nLocal data file location : " + ingredientBookFilePath);
+
+
         sb.append("\nLocal data file location : " + salesBookFilePath);
         sb.append("\nLocal data file location : " + ingredientBookFilePath);
 
+
         return sb.toString();
     }
 
```

Comment:
There are a lot of empty lines in this java file. Are they all necessary and can it be cleaned up?